corehq/apps/app_manager/urls.py

```diff
@@ -84,7 +84,6 @@
 
 
 urlpatterns = [
-    url(r'^$', view_app, name='default_app'),
```

Comment:
what happens to a user if they try to go to `<server>/a/<domain>/apps/` after removing this pattern?

Reply:
They'll get a 404. There aren't any direct links to this (anymore), and I don't think it's possible to save that url as a bookmark unless you manually type it in (since it's currently a redirect).